dashboard/modules/snapshot/snapshot_head.py

```diff
@@ -14,14 +14,17 @@ def __init__(self, dashboard_head):
         super().__init__(dashboard_head)
         self._gcs_job_info_stub = None
         self._gcs_actor_info_stub = None
+        self._dashboard_head = dashboard_head
 
     @routes.get("/api/snapshot")
     async def snapshot(self, req):
         job_data = await self.get_job_info()
         actor_data = await self.get_actor_info()
+        session_name = await self.get_session_name()
         snapshot = {
             "jobs": job_data,
             "actors": actor_data,
+            "session_name": session_name,
```

Comment:
schema uses camelCase (`sessionName`), but this is using snake_case. please double check.